build.sh

```diff
@@ -89,6 +89,15 @@ cd -
 
 rm -rf ffbuild
 
+cd "${ARTIFACTS_PATH}" &&
+{
```

Comment:
Use can use a subshell:
```bash
(
    cd "${ARTIFACTS_PATH}"
    ...
    # unset and cd - are no longer necessary since they won't propagate to the parent anyway.
)
```

Reply:
This is true.